.changeset/calm-bulldogs-repair.md

```diff
@@ -1,6 +1,5 @@
 ---
 '@commercetools-docs/gatsby-theme-docs': minor
-'@commercetools-docs/ui-kit': patch
 ---
 
-The RssFeed is now one single table which shows the newest release notes of all feeds
```

Comment:
This is a different changeset, you should not change this.

Reply:
Sry for that, I accidently changed the wrong changeset...thanks 👍 
d038274